app/assets/scss/app.scss

```diff
@@ -1,5 +1,6 @@
 $govuk-show-breakpoints: true;
 
-@import "govuk-frontend/all.scss";
-@import "./partials/app";
-@import "./partials/banner";
+@import 'govuk-frontend/all.scss';
```

Comment:
We should try and be consistent about using either single quotes or double quotes in Sass files; `src/digitalmarketplace/_all.scss` uses double quotes